### PR TITLE
fixed the bug for restore meta.

### PR DESCRIPTION
**Pull request by monadbobo**
1. Fix the bug of incorrectly calling storage's interface when restore meta (no need to copy to follower node).

2. add the interface to write storage directly to Nebulastore.

### CODE REVIEW

File: src/kvstore/KVStore.h [hunk 1]
```diff
@@ -208,6 +208,9 @@ class KVStore {
     virtual ResultCode restoreFromFiles(GraphSpaceID spaceId,
                                         const std::vector<std::string>& files) = 0;
 
+    virtual ResultCode multiPutWitoutReplicator(GraphSpaceID spaceId,
```
Comment (bright-starry-sky) [inline src/kvstore/KVStore.h:211]: `multiPutWithoutReplicator`

**Comment by critical27** [comment]
Em, I am not pretty sure if this PR will effect other function. As far as I know, the http replace interface do need to write to all meta.

**Comment by monadbobo** [comment]
> Em, I am not pretty sure if this PR will effect other function. As far as I know, the http replace interface do need to write to all meta.

The current http interface does not use replaceHostInPartition, so perhaps we can add a parameter to replaceHostInPartition to control whether it needs to take the draft protocol.

**Review by critical27 — APPROVED** [review]

**Review by panda-sheep — APPROVED** [review]
Good，Excellent！

**Pull request merged**